### PR TITLE
[HMC-62] - Layout Dashboard

### PR DESCRIPTION
**Pull request by mpsalunggg**
## Description :

-   Create Navbar
-   Create Sidebar
-   Create Darkmode

## Screenshots / Video Record:

https://github.com/hammer-code/lms-fe/assets/92923869/be534ce6-604f-41f8-88eb-9685b5c05303

### CODE REVIEW

File: src/providers/sidebarProvider.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+'use client';
+import React, { createContext, ReactNode, useState } from 'react';
+
+export interface SidebarContextProps {
```
Comment (vickyadrii) [inline src/providers/sidebarProvider.tsx:4]: i'd prefer to move it to folder `types`.

Comment (mpsalunggg) [reply src/providers/sidebarProvider.tsx]: @vickyadrii done kaks

**Review by vickyadrii — APPROVED** [review]

File: src/components/common/ThemeProvider/index.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+'use client';
```
Comment (akwanmaroso) [inline src/components/common/ThemeProvider/index.tsx:1]: is this necessary in the context ?

Comment (mpsalunggg) [reply src/components/common/ThemeProvider/index.tsx:1]: @akwanmaroso I dont think so, do I just delete it?

Comment (akwanmaroso) [reply src/components/common/ThemeProvider/index.tsx:1]: If it's not needed, just remove it

**Review by akwanmaroso — APPROVED** [review]

**Pull request merged**